runner/execer/os/os_execer.go

```diff
@@ -89,21 +90,35 @@ func (p *osProcess) monitorMem(memCap execer.Memory, stat stats.StatsReceiver) {
 	reportThresholds := []float64{0, .25, .5, .75, .85, .9, .93, .95, .96, .97, .98, .99, 1}
 	memTicker := time.NewTicker(10 * time.Millisecond)
```

Comment:
This seems too frequent for running a ps and starting a python interpreter and could be a cpu distraction - do we think this wouldn't be as effective if it was 100ms or even 1s?